docs/strategies/fallback.md

```diff
@@ -97,7 +97,9 @@ sequenceDiagram
     P->>F: Calls ExecuteCore
     F->>+D: Invokes
     D->>-F: Fails
-    F->>F: Falls back to<br/>substitute result
+    activate F
+    F-->>F: Falls back to<br/>substitute result
+    deactivate F
```

Comment:
Should we illustrate where the substitute result comes from?

Reply:
Something like
```
participant FA as FallbackAction
...

F->>+FA: Invokes
FA-->>FA: Calculates substitute result
FA->>-F: Returns substituted result
```
?